### DISCUSSION TITLE
Transfer stuck

### DISCUSSION THREAD

**Discussion by flyinginger** [aidenlx/marginnote-companion#28] [Q&A]
Hello and thanks for this so helpfull bridge.

In MN3 bridge open
In obsidian auto paste on

In MN3 When in the map mode to transfer toward OBS some exerpt just do not want to copy over.
Others still and are transfering, but some exception just do not copy over...

Reinstalling both plug-in solved temporarly the issue.

I transfer alot... ;)

**Comment by aidenlx** [comment]
System, MarginNote version, obsidian-bridge and marginnote-companion plugin version and some screenshot on the excerpt that won't be transferred? Can the problem be reproduced now? If you can, it's best to send a screen recording when problem occurred.
